include/taskolib/Sequence.h

```diff
@@ -254,6 +254,8 @@ class Sequence
      *
      * During execute(), is_running() returns true to internal functions or LUA callbacks.
      *
+     * By executing the Context the step setup script is overwritten.
```

Comment:
... you mean, by executing the sequence?

Reply:
Oh man! Mistakes like that come when I am in a hurry. Changing to `... executing a Sequence ...`.